sdk/src/types/block/protocol/mod.rs

```diff
@@ -263,6 +263,62 @@ impl ProtocolParameters {
             max: self.max_committable_age(),
         }
     }
+
+    /// Calculates the past bounded slot for the given slot of the SlotCommitment.
+    /// Given any slot index of a commitment input, the result of this function is a slot index
+    /// that is at least equal to the slot of the block in which it was issued, or higher.
+    /// That means no commitment input can be chosen such that the index lies behind the slot index of the block,
+    /// hence the past is bounded.
+    pub fn past_bounded_slot(&self, slot_commitment_id: SlotCommitmentId) -> SlotIndex {
```

Comment:
I just prefer using this syntax
```rust
protocol_parameters.past_bounded_slot(slot_commitment_id)
```
rather than this
```rust
slot_commitment_id.past_bounded_slot(protocol_parameters.max_commitable_age())
```
and it matches the other usages as well

Reply:
Well if you insist. I don't think we need convenience on such specific and low level computations at all. Just creates maintenance burden to duplicate.